website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: "@/application-shell-connectors"
+---
+
+<Subtitle>
+
+Components to work with the `@commercetools-frontend/application-shell` package to access and subscribe to specific data within the AppShell.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/application-shell-connectors
+
+# or
+
+npm --save install @commercetools-frontend/application-shell-connectors
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react @apollo/client
+
+# or
+
+npm --save install react @apollo/client
+```
+
```

Comment:
After a first look, I think I would probably imagine the structure of the page to be something like this:

```
# Application context

## Application properties

### `environment`

### `user`

### `project`

### `dataLocale`

## Custom user properties

## Usage

### useApplicationContext

### withApplicationContext

# Project image settings

## Usage

### GetProjectExtensionImageRegex

### withProjectExtensionImageRegex
```

Reply:
Structure updated in this commit 80980a4caec6d6f8973abd3475b4744daaa21e6a